meta-ros-common/classes/ros_cmake.bbclass

```diff
@@ -13,3 +13,8 @@ EXTRA_OECMAKE:prepend:class-native = "\
     -DCMAKE_PREFIX_PATH='${STAGING_DIR_NATIVE}${ros_prefix};${STAGING_DIR_NATIVE}${prefix}' \
     -DCMAKE_INSTALL_PREFIX:PATH='${ros_prefix}' \
 "
+
+EXTRA_OECMAKE:prepend:class-nativesdk = "\
+    -DCMAKE_PREFIX_PATH='${STAGING_DIR_NATIVE}/opt/ros/${ROS_DISTRO};${STAGING_DIR_NATIVE}${ros_prefix};${STAGING_DIR_NATIVE}${prefix}' \
```

Comment:
Use ${ros_prefix} to replace opt/ros/${ROS_DISTRO}

Reply:
Thanks! I will test and feedback. Sorry for the delay, my attention has been distracted.

Reply:
This change still allows the SDK to build and run, but I suspect it is not used in my case.